aspnet/signalr/overview/performance/scaleout-with-redis.md

```diff
@@ -130,15 +130,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
 
 Add the IIS role. Include "Application Development" features, including the WebSocket Protocol.
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'WebSocket Protocol' is selected.](scaleout-with-redis/_static/image5.png)
+![Screenshot that shows a window titled Add Roles and Features Wizard On the Server Roles option WebSocket Protocol is selected.](scaleout-with-redis/_static/image5.png)
 
 Also include the Management Service (listed under "Management Tools").
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'Management Service' is selected.](scaleout-with-redis/_static/image6.png)
+![Screenshot that shows a window titled Add Roles and Features Wizard On the Server Roles option Management Service is selected.](scaleout-with-redis/_static/image6.png)
 
 **Install Web Deploy 3.0.** When you run IIS Manager, it will prompt you to install Microsoft Web Platform, or you can [download the installer](https://go.microsoft.com/fwlink/?LinkId=255386). In the Platform Installer, search for Web Deploy and install Web Deploy 3.0
 
-![Window titled 'Web Platform Installer 4.5'. 'Web Deploy 3.0' is selected.](scaleout-with-redis/_static/image7.png)
+![Screenshot that shows a window titled Web Platform Installer 4 point 5 Web Deploy 3 point 0 is selected.](scaleout-with-redis/_static/image7.png)
```

Comment:
How about, "Screenshot that shows Web Platform Installer 4 point 5 dialog box. Web Deploy 3 point 0 is selected."